frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "ec2:DescribeInstances",
+                "ec2:DescribeRouteTables",
+                "ec2:DescribeInstanceTypeOfferings",
+                "ec2:DescribeVpcEndpoints"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "s3:List*",
+                "s3:Get*",
+                "s3:DeleteObject",
+                "s3:PutBucketNotification"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:s3:::*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:UpdateNotebookInstanceLifecycleConfig"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance-lifecycle-config/*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "Null": {
+                    "aws:ResourceTag/project": "false",
+                    "aws:ResourceTag/system": "false",
+                    "aws:ResourceTag/user": "false"
+                }
+            },
+            "Action": [
+                "sagemaker:DeleteNotebookInstance",
+                "sagemaker:DescribeNotebookInstance",
+                "sagemaker:StopNotebookInstance"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:DeleteNotebookInstanceLifecycleConfig",
+                "sagemaker:DescribeNotebookInstanceLifecycleConfig"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance-lifecycle-config/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:ListNotebookInstanceLifecycleConfigs",
+                "sagemaker:ListNotebookInstances"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": "lambda:InvokeFunction",
+            "Resource": "arn:{AWS_PARTITION}:lambda:{AWS_REGION}:{AWS_ACCOUNT}:function:mls-lambda-*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:ListClusters",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:DescribeCluster",
+                "elasticmapreduce:ListInstances",
+                "elasticmapreduce:TerminateJobFlows",
+                "elasticmapreduce:SetTerminationProtection"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:elasticmapreduce:{AWS_REGION}:{AWS_ACCOUNT}:cluster/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
```

Comment:
So I take it the goal here is that they don't attach the notebook policy either? If you are planning on doing a single policy, then you will also need all the stop, list, and describe permissions from our notebook policy added in this policy.

I like the idea of giving the customer the option to choose convenience of reusing the notebook and app policy or using least-privileges security by creating this role. Additionally, if they choose to re-use app policy and notebook policy it reduces need for duplicate updates in the future if they are needed.

TLDR:
- If reusing notebook policy remove this
- If consolidating notebook policy permissions, I think permissions related to other services are missing

Reply:
Went through and added all of the notebook policies that were missing in this policy. Excluded stuff that obviously wasn't applicable. Also updated the instructions to have a caveat that you could either create this new policy, or re-use the app policy + notebook policy.

Reply:
Final sanity check, but can you try creating a policy with all of the combined statements as a sanity check that this is a valid length.